### DISCUSSION TITLE
How can I do a console.log() from the front end?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#55825] [Help] [answered]
### Summary

This code:

```
const Testing = () => {
  const logMe = () => {
    console.log("testing it");
  };

return(
<Col>
          <button className="btn-send" onClick={logMe}>
            Send
          </button>
        </Col>
);
};

export default Testing;
```

^^^  This is just a test... at times I need to do console.log from the front end just to make sure a function is responding.....  but there is nothing showing on the console when I do something like that..... HOWEVER, other functions still seem to run as they should, but they skip the console.log....  How can I fix that?

**Comment by Elindo586** [comment]
Ok... it is showing in the browser console... BUT not in my VS Code terminal?.... why is that??

**Comment by devjiwonchoi** [comment, marked as answer]
By default, the Components in Next.js are Server Components, which lets the console.log to be logged on the server.
If you've managed to log on the browser, the Component is rendered on the client-side, which makes the console.log code to be managed also on the client side, resulting not appearing on your server (localhost, console appeared to your VS Code).

**Reply by Elindo586** [reply]
Thank you!
